spec/models/spree/product_spec.rb

```diff
@@ -168,7 +168,7 @@ module Spree
       end
 
       it "requires a unit value" do
-        expect(build(:simple_product, unit_value: nil)).not_to be_valid
+        expect(build(:simple_product, unit_value: nil)).to be_valid
```

Comment:
The spec says: "requires a unit value" and then the content now does the opposite. I think that we need to spec both cases, the case where it's required and the case where it's not.